sdk/python/feast/client.py

```diff
@@ -493,17 +497,70 @@ def get_feature_set(
             raise grpc.RpcError(e.details())
         return FeatureSet.from_proto(get_feature_set_response.feature_set)
 
-    def list_entities(self) -> Dict[str, Entity]:
+    def list_features(
+        self,
+        project: str = None,
+        entities: List[str] = list(),
+        labels: Dict[str, str] = dict(),
+        return_as_dict=True,
+    ) -> Dict[FeatureRef, Feature]:
         """
-        Returns a dictionary of entities across all feature sets
+        Returns a list of features based on filters provided
+
+        Returns:
+            Dictionary of <feature references: features>
+        """
+        self._connect_core()
+
+        if project is None:
+            if self.project is not None:
+                project = self.project
+            else:
+                project = "*"
```

Comment:
I want to eliminate all these wildcards in the code because it adds unnecessary complexity.